src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -157,48 +160,81 @@ private static string[] GetDefaultLetterKeyAllLanguagesOnly(LetterKey letter)
+                LetterKey.VK_Z => new[] { "ʒ", "ǯ" },
+                LetterKey.VK_COMMA => new[] { "₋", "⁻", "–" }, // – is in VK_MINUS for other languages, but not VK_COMMA, so we add it here.
                 LetterKey.VK_PERIOD => new[] { "…", "\u0300", "\u0301", "\u0302", "\u0303", "\u0304", "\u0308", "\u030B", "\u030C" },
-                LetterKey.VK_MINUS => new[] { "~", "‐", "‑", "‒", "—", "―", "⁓", "−", "⸺", "⸻", "∓" },
-                LetterKey.VK_SLASH_ => new[] { "÷", "√" },
-                LetterKey.VK_DIVIDE_ => new[] { "÷", "√" },
-                LetterKey.VK_MULTIPLY_ => new[] { "×", "⋅" },
-                LetterKey.VK_PLUS => new[] { "≤", "≥", "≠", "≈", "≙", "⊕", "⊗", "∓", "≅", "≡" },
+                LetterKey.VK_MINUS => new[] { "‐", "‑", "‒", "—", "―", "⁓", "−", "⸺", "⸻" },
+                _ => Array.Empty<string>(),
+            };
+        }
+
+        // Mathematics
+        private static string[] GetDefaultLetterKeyMATH(LetterKey letter)
+        {
+            return letter switch
+            {
+                LetterKey.VK_0 => new[] { "₀", "⁰", "°", "∅", "↉" },
+                LetterKey.VK_1 => new[] { "₁", "¹", "½", "⅓", "¼", "⅕", "⅙", "⅐", "⅛", "⅑", "⅒" },
+                LetterKey.VK_2 => new[] { "₂", "²", "⅔", "⅖", "√" },
+                LetterKey.VK_3 => new[] { "₃", "³", "¾", "⅗", "⅜", "∛" },
+                LetterKey.VK_4 => new[] { "₄", "⁴", "⅘", "∜" },
+                LetterKey.VK_5 => new[] { "₅", "⁵", "⅚", "⅝" },
+                LetterKey.VK_6 => new[] { "₆", "⁶" },
+                LetterKey.VK_7 => new[] { "₇", "⁷", "⅞" },
+                LetterKey.VK_8 => new[] { "₈", "⁸", "∞", "∝" },
+                LetterKey.VK_9 => new[] { "₉", "⁹" },
+                LetterKey.VK_A => new[] { "α", "∀" },
+                LetterKey.VK_B => new[] { "β" },
+                LetterKey.VK_C => new[] { "χ", "ℂ", "∁" },
+                LetterKey.VK_D => new[] { "δ", "∂", "◇" },
+                LetterKey.VK_E => new[] { "ε", "η", "∃", "∄", "∈", "∉" },
+                LetterKey.VK_F => new[] { "φ", "ƒ" },
+                LetterKey.VK_G => new[] { "γ" },
+                LetterKey.VK_I => new[] { "ι" },
+                LetterKey.VK_K => new[] { "κ" },
+                LetterKey.VK_L => new[] { "λ" },
+                LetterKey.VK_M => new[] { "μ" },
+                LetterKey.VK_N => new[] { "ν", "ⁿ", "ℕ" },
+                LetterKey.VK_O => new[] { "ο", "ω" },
+                LetterKey.VK_P => new[] { "π", "ψ", "φ", "∏", "⊥", "⊢", "⊬" },
+                LetterKey.VK_Q => new[] { "ℚ" },
+                LetterKey.VK_R => new[] { "ρ", "ℝ" },
+                LetterKey.VK_S => new[] { "σ", "∑", "∫", "∮" },
+                LetterKey.VK_T => new[] { "θ", "τ", "⊨", "⊭" },
+                LetterKey.VK_U => new[] { "υ" },
+                LetterKey.VK_X => new[] { "ξ", "×", "⋅" },
+                LetterKey.VK_Y => new[] { "υ" },
+                LetterKey.VK_Z => new[] { "ζ", "ℤ" },
```

Comment:
Hi @octastylos-pseudodipteros, in the issue's comments, there were suggestions of not including the Greek alphabets as they are already included. Any reason for specifically including them here?

Reply:
Greek letters are often used together with math symbols and even though there is a separate language for them, it is not possible to use both math and Greek letters language at the same time (if the user doesn't want to have symbols from all languages).

Reply:
A couple of issues with this understanding - 
1. The issue you're stating is open #21261. It is only a matter of time before it gets implemented and this issue is resolved.
2. If in the future, we have to change some symbols / add some (symbols/accents) in the Greek language, we would have to deal with it in two places (redundancy).

My suggestion would be to remove the Greek references from your function and pass the letter to the `GetDefaultLetterKeyEL` function to get the Greek references for that particular letter. You could then concatenate these in your string array and return the same.

Reply:
That is a good idea, I will do it. Thank you

Reply:
Hello @GhostVaibhav, in the new commit Math language uses Greek letters from the Greek letter alphabet, but because of this there are also letters with tonos (eg. ά, έ...), which aren't commonly used in math.

Reply:
Hi @octastylos-pseudodipteros, that should not be a major blocking issue since all the other characters would be legally used. I have other concerns with removing "π" from Portuguese.

<hr>

@htcfreek the "π" symbol doesn't have any special meaning in Portuguese and this PR removes it from there. Is it allowed to change the base definitions of already defined languages?

Reply:
@htcfreek thoughts?

Reply:
Sory for the late reply. I missed your question.

I think removing is okay. If the character doesn't belongs to a language it makes no sense to have it in the set.

Reply:
Hi @htcfreek, the PR looks good to me. Can you also review it (just in case)?